src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -75,9 +70,9 @@ public class HashSet<T> : ICollection<T>, ISet<T>, IReadOnlyCollection<T>
 
         #region Constructors
 
-        public HashSet()
-            : this(EqualityComparer<T>.Default)
-        { }
+        public HashSet() : this(EqualityComparer<T>.Default)
```

Comment:
I don't really care for this sort of change. It doesn't really go against our style guide, except for the braces being on the same line.